api/envoy/config/listener/v3/listener_components.proto

```diff
@@ -165,9 +166,25 @@ message FilterChainMatch {
   repeated string application_protocols = 10;
 }
 
+// The configuration for on-demand filter chain. If this field is not empty in FilterChain message,
+// a filter chain will be built on-demand.
+// On-demand filter chains will be first built as placeholders when a listener is created in order to
+// speedup the warming up of the listener. The building and initialization of an on-demand filter chain
+// will be postponed to the arrival of new connections that require this filter chain.
+// The building process of on-demand filter chains will be similar to the one of normal filter chains.
+// However, a unique rebuilder with a unique init manager will be assigned to build each on-demand filter chain.
```

Comment:
remove "init manager" and make this more end user focused please

Reply:
ok~